plugins/frontend/backstage-plugin-datadog/src/components/GraphWidget.tsx

```diff
@@ -15,8 +15,10 @@
  */
 
 import { Entity } from '@backstage/catalog-model';
-import { useEntity } from '@backstage/plugin-catalog-react';
-import { MissingAnnotationEmptyState } from '@backstage/core-components';
```

Comment:
MissingAnnotationEmptyState from core-components is deprecated / shifted to importing the same thing from plugin-catalog-react instead